components/Nav.js

```diff
@@ -60,6 +69,17 @@ export default ({ file }) => {
       <p>
         ${description || 'There is no description for this package.'}
       </p>
+      <h2>Versions</h2>
+      <select id="version" onChange=${e => changeVersion(e.target.value)}>
```

Comment:
should we give this a value of `selectedVersion`?

At the moment, on first load of a package, you see the minimum version in the `select`, rather than the version you're looking at (always "latest" I guess)

Reply:
Update: latest stable release, I think (i.e. not alpha/beta)